command/base_helpers.go

```diff
@@ -296,13 +296,26 @@ func parseFlagFile(raw string) (string, error) {
 func generateFlagWarnings(args []string) string {
 	var trailingFlags []string
 	for _, arg := range args {
-		if strings.HasPrefix(arg, "-") {
-			trailingFlags = append(trailingFlags, arg)
+		if !strings.HasPrefix(arg, "-") {
+			continue
 		}
+
+		isEnvFlag := false
```

Comment:
I can see what it's getting at, but "environmental flag" feels a bit unclear to me, especially in the context of flag/option parsing where it could be confused to mean "flags that use environment variables" in some way. Would global be a better adjective to use? i.e. they are flags that can appear anywhere in the command line for almost any command, so they're global flags.

Reply:
I agree globalFlag to denote flags that apply to any command might be more clear. I'll update the naming.